src/styles/Document.ts

```diff
@@ -4,3 +4,8 @@ cssRule('html, body', {
   margin: 0,
   padding: 0,
 });
+
+//to prevent the vertical overflow of the root element
```

Comment:
The comment you added paraphrase the CSS rule, so it's not super useful in its current shape.
A useful comment will have  a _why_ or _when_. Can you add in which case this happens? What is the result to the user if we don't have this rule?

I sincerely don't understand why we need this :-) When I remove it I don't see a difference.

Reply:
<img width="1057" alt="Screenshot 2024-05-02 at 08 50 40" src="https://github.com/mozilla/perfcompare/assets/88336547/cf13c68e-9bd6-4786-a607-6ec4511f75c2">

Honestly, it's just reducing an excess of vertical flow to the page (the scroll bar responds appropriately, scrolling further down). I know that's repeating myself but I'm not sure how else to say it. But if I can't convince you, I can remove it and just update the body background color, etc. I don't know why it ended up this gray color in the first place.